src/services/email.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports.send = async (event) => {
```

Comment:
Looking into changing this to a class

Reply:
Same comment here, was gonna ask to change the name to `send_mail`